core/templates/base-components/oppia-root.directive.ts

```diff
@@ -117,28 +117,14 @@ angular.module('oppia').directive('oppiaRoot', [
               );
             }
 
-            // TODO(#12793): Remove the use of (
-            // OppiaAngularRootComponent.ajsTranslate).
-            OppiaAngularRootComponent.ajsTranslate = $translate;
-            const translateService = (
-              OppiaAngularRootComponent.translateService);
-            const translateCacheService = (
-              OppiaAngularRootComponent.translateCacheService);
             const i18nLanguageCodeService = (
               OppiaAngularRootComponent.i18nLanguageCodeService);
 
             i18nLanguageCodeService.onI18nLanguageCodeChange.subscribe(
               (code) => {
-                translateService.use(code);
```

Comment:
Both `oppia-root` and `oppia-angular-root` directives/components will present in a hybrid page (page using both angular and angularjs). But an angular page will only have `oppia-angular-root` component.

So, I moved some translation code from here to the `oppia-angular-root component`.